### DISCUSSION TITLE
how can i send msg in the theread i just create??

### DISCUSSION THREAD

**Discussion by mansan1230** [Rapptz/discord.py#7716] [Q&A] [answered]
sorry for a stupid question, 
how can i send msg in the theread i just create??
![image](https://user-images.githubusercontent.com/84826972/159046594-1fce199a-f5df-43b7-b93c-e65ed7d57cf6.png)

**Comment by LostLuma** [comment, marked as answer]
The [`create_thread`](https://discordpy.readthedocs.io/en/master/api.html#discord.TextChannel.create_thread) method returns the new thread, meaning you can assign it to a variable and then send messages or do other operations on it:

```py
thread = await channel.create_thread(...)
await thread.send(...)
```
